### PR TITLE
I44 create about us page

### LINKED ISSUE #44: Create About us Page
## Basic Information
- A static page with assets from google drive
- Follow figma design
- If you feel you can add to the page by adding animations go for it! (Not required though)
- Build for mobile first, we will do the designs for desktop later!

### PR DESCRIPTION
**Pull request by ChantelleKerr**
## Change Summary
- Added About us page
- Added a footer component
- Uses CSF fonts and set the default font to Sophia Pro
- Images and text are subject to change after clients feedback
- Donate link will be given to us by client

- Roast me Drew and Frinze

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
Resolve #44 

# Related Issue

- Resolve #44

### CODE REVIEW

File: client/src/views/AboutView.vue [hunk 1]
```diff
@@ -1,15 +1,145 @@
...
+    Foundation's journey of helping to support First Nations young people to realise their potential
+    through lifelong learning.
+  </p>
+  <!-- TODO: Use CSF for button -->
+  <v-row justify="end" class="px-8 pb-4" no-gutters>
+    <v-btn
+      class="bg-primaryRed pb-14"
+      size="x-large"
+      variant="flat"
+      :style="{ fontFamily: 'Hackney', fontSize: '42px' }"
```
Comment (Drewbi) [inline client/src/views/AboutView.vue:40]: I know there will be a better way of styling every button with the same font but I can't find a solid answer for it.

**Review by Drewbi — CHANGES_REQUESTED** [review]

**Comment by Drewbi** [comment]
The main change for this is moving the footer in the shared `App.vue` everything else is just suggestions. Also I'd suggest making adding a screenshot of your frontend work to your PR a standard process in the team, makes reviewing a lot easier.

Some of the changes are style comments and so might be more applicable to the figma team if you're just implementing their designs

File: client/src/views/AboutView.vue [hunk 2]
```diff
@@ -1,15 +1,145 @@
...
+  <v-img :src="thankYou" width="100%"></v-img>
+  <v-row class="ma-8" no-gutters>
+    <p>
+      The Community Spirit Foundation acknowledges and pays our respects to the past, present and
+      emerging traditional custodians of the land on which we work and live. Aboriginal and Torres
+      Strait Islander users are advised that this website may contain images and voices of people
+      who have died.
+    </p></v-row
+  >
+  <Footer />
```
[the remark this answers is not in the document]
Comment (ChantelleKerr) [reply client/src/views/AboutView.vue:111]: I'm not sure if I want it on every page yet

File: client/src/components/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      <v-col cols="auto">
+        <v-icon color="white" class="pr-3">mdi-email</v-icon>
+        <a class="text-primaryWhite">admin@communityspiritfoundation.org.au</a>
+      </v-col>
+    </v-row>
+    <v-row align="center" no-gutters class="pl-6">
+      <v-icon color="white" size="small" class="pr-4">mdi-phone</v-icon>
+      <p class="text-primaryWhite">(03) 9686 4813</p>
+    </v-row>
+    <v-row align="center" justify="end" no-gutters class="mr-1">
```
[the remark this answers is not in the document]
Comment (ChantelleKerr) [reply client/src/components/Footer.vue:14]: From looking at it again, I sorta like it at the bottom

**Comment by ChantelleKerr** [comment]
Updated navbar as well
Screen Shot 2023-07-07 at 8 00 25 pm

File: client/src/components/Footer.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (nisarg-dave) [reply client/src/components/Footer.vue:14]: I agree, looks better in the bottom right.

File: client/src/views/AboutView.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (nisarg-dave) [reply client/src/views/AboutView.vue:113]: Could you please list out the pages you told me you think it should be on?

File: client/src/components/Footer.vue
[the remark this answers is not in the document]
Comment (nisarg-dave) [reply client/src/components/Footer.vue:1]: Yeah that's strange. It disappeared when you only use v-footer for me.

File: client/src/views/AboutView.vue (hunk 1 above)
Comment (nisarg-dave) [reply client/src/views/AboutView.vue:39]: You could use https://vuetifyjs.com/en/styles/text-and-typography/ typography helper classes to control size of text but if this is applied then the font-family doesn't apply and it defaults to the Material Design font, Roboto.

**Comment by nisarg-dave** [comment]
> Updated navbar as well Screen Shot 2023-07-07 at 8 00 25 pm

Could you please explain what was updated?

**Comment by ChantelleKerr** [comment]
> > Updated navbar as well Screen Shot 2023-07-07 at 8 00 25 pm
> 
> Could you please explain what was updated?

It was a navbar of buttons. It also wasnt sticky

Screen Shot 2023-07-08 at 1 36 26 pm

File: client/src/views/AboutView.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (ChantelleKerr) [reply client/src/views/AboutView.vue:113]: maybe challenges and events

**Review by nisarg-dave — APPROVED** [review]
Looks good.

**Pull request merged**